Mathlib/Data/Polynomial/Basic.lean

```diff
@@ -889,6 +891,9 @@ theorem X_pow_eq_monomial (n) : X ^ n = monomial n (1 : R) := by
   · rw [pow_succ', hn, X, monomial_mul_monomial, one_mul]
 #align polynomial.X_pow_eq_monomial Polynomial.X_pow_eq_monomial
 
+@[simp] lemma coeff_pow_X {m n : ℕ} : coeff (X^n) m = if n = m then 1 else 0 := by
```

Comment:
I think that `pow` and `X` should be swapped and also that `Polynomial.coeff_X_pow` exists, right?